### PR TITLE
Update attack-surface-reduction-rules-reference.md

### PR DESCRIPTION
**Pull request by msbemba**
Add a note that the PS scripts have been temporarily excluded from the "Block execution of potentially obfuscated scripts" rule  due to the large scale FP issues faced in the past.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8506

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8530

File: microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-reference.md [hunk 1]
```diff
@@ -325,6 +325,9 @@ Dependencies: MDAV, Cloud Protection
 ### Block execution of potentially obfuscated scripts
 
 This rule detects suspicious properties within an obfuscated script.
+  
+> [!IMPORTANT]
+> The Powershell scripts have been temporarily excluded from the "Block execution of potentially obfuscated scripts" rule  due to the large scale FP issues faced in the past.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-reference.md]:
```suggestion
> PowerShell scripts have been temporarily excluded from the "Block execution of potentially obfuscated scripts" rule due to the large-scale FP issues faced in the past.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR.

**Pull request merged**